synth-converter/src/parser/actions.rs

```diff
@@ -0,0 +1,93 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize)]
+#[allow(non_snake_case)]
```

Comment:
suggestion: Clean and good that you put it in an own file!
Nice that you found away around this, which is something good to know.
However I am strongly against doing that cause of the following reasons:

- Code adhers to the style guide and best practices (no excuse). That includes the formatting rules for Rust in `rustfmt`.
- Suddenly code is mixed now, it contains non standard naming together with standard naming. This is not good.
- I understand you did that to have less boilerplate: This should not matter IMO. Because what do you gain? Some lines of code? If you want you can do a macro `define_field` (cannot recite right away) which writes you 2 lines `#[serde(rename = "asdf")] ...`, to make it less verbose. We can have a look at it ;)

Reply:
Okay so you mean I should rename almost every field? The thing here is that the input was named to match the ontology and such the output. So for every field it would be like this:
`input -> rs -> output`: `startTime -> starting_time -> startTime`
Let's talk about that later.